### PR TITLE
[feat] Set up system for locale-specific help

### PR DESCRIPTION
**Pull request by toddself**
Provide a way to determine the locale of the user and load in a
localized help file, defaulting to english.

Add example help file from #111

### CODE REVIEW

**Comment by zacanger** [comment]
@toddself your build may be broken because I had some bugs in a commit, if you pull from upstream master I think that'll fix that.

File: cli/lib/commands/login.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const loginOpts = figgy({
 });
 
 async function login(opts) {
+  console.log(opts);
```
Comment (chrisdickinson) [inline cli/lib/commands/login.js]: EX-TERM-IN-ATE

**Comment by toddself** [comment]
thanks @zacanger i rebased off master.  @chrisdickinson that console.log has been taken out!

**Comment by chrisdickinson** [comment]
Yay, awesome work. Merging now!

**Pull request merged**